### PR TITLE
#1961 resolve [Core] add channel var rtp_2833_payload to overwrite rfc2833-pt in sip profile

### PR DESCRIPTION
**Pull request by mahald**
Problem with rfc2833-pt and DTMF on bridged calls ever since. New channel variable rtp_2833_payload to set default dmtf payload on outbound calls

### CODE REVIEW

File: src/switch_core_media.c [hunk 1]
```diff
@@ -10408,6 +10408,12 @@ SWITCH_DECLARE(void) switch_core_media_gen_local_sdp(switch_core_session_t *sess
 			for (i = 0; i < smh->num_rates; i++) {
 				if (smh->rates[i] == 8000 || smh->num_rates == 1) {
 					smh->dtmf_ianacodes[i] = smh->mparams->te;
+
+					if (!zstr(switch_channel_get_variable(session->channel, "rtp_2833_payload"))) {
```
Comment (andywolk) [inline src/switch_core_media.c:10412]: Can we get this once?

**Comment by mahald** [comment]
ok done

**Comment by mahald** [comment]
Are you going to merge this pull request? As far as i understand the failing notify has nothin to do with the code.

**Comment by mahald** [comment]
I don't see what test acutally fail but it seems not be related to this work.

**Pull request closed without merging**